packages/paste-website/src/pages/form-elements/textarea/index.mdx

```diff
@@ -0,0 +1,598 @@
+        </UnorderedList>
+      </td>
+      <td>No</td>
+    </tr>
+    <tr>
+      <td>Prefix/suffix text</td>
+      <td>
+        <UnorderedList marginBottom="space0">
+          <ListItem>Default: $color-text</ListItem>
+          <ListItem>Disabled: $color-text-weak</ListItem>
+        </UnorderedList>
+      </td>
+      <td>No</td>
+    </tr>
+    <tr>
+      <td>Valut text</td>
+      <td>
+        <UnorderedList marginBottom="space0">
+          <ListItem>$font-size-30</ListItem>
+          <ListItem>Default, Readonly: $color-text</ListItem>
+          <ListItem>Disabled: $color-text-weak</ListItem>
+        </UnorderedList>
+      </td>
+      <td>No, but it can take any children</td>
+    </tr>
+    <tr>
+      <td>Placeholder text</td>
+      <td>
+        $color-text-weak, italic
+      </td>
+      <td>No</td>
+    </tr>
+    <tr>
+      <td>Help text</td>
+      <td>
+        $color-text-weak, $font-size-30
+      </td>
+      <td>No</td>
+    </tr>
+    <tr>
+      <td>Inline error</td>
+      <td>
+        <UnorderedList marginBottom="space0">
+          <ListItem>Text: $color-text-error, $font-size-30</ListItem>
+          <ListItem>Icon: IconError, $color-text-error, $icon-size-30</ListItem>
+        </UnorderedList>
+      </td>
+      <td>No</td>
+    </tr>
+    <tr>
+      <td>Spacing</td>
+      <td>
+        <UnorderedList marginBottom="space0">
+          <ListItem>Label:
+            <UnorderedList marginBottom="space0">
+              <ListItem>Bottom: $space-10</ListItem>
+              <ListItem>Between label and required indicator: $space-20</ListItem>
+            </UnorderedList>
+          </ListItem>
+          <ListItem>Textarea:
+            <UnorderedList marginBottom="space0">
+              <ListItem>Left padding: $space-40</ListItem>
+              <ListItem>Top, right, bottom padding: $space-30</ListItem>
+            </UnorderedList>
+          </ListItem>
+          <ListItem>Prefix/suffix padding: $space-30</ListItem>
+          <ListItem>Help text:
+            <UnorderedList marginBottom="space0">
+              <ListItem>Top: $space-20</ListItem>
+            </UnorderedList>
+          </ListItem>
+          <ListItem>Inline error:
+            <UnorderedList marginBottom="space0">
+              <ListItem>Spacing between icon and text: $space-20</ListItem>
+              <ListItem>Top: $space-20</ListItem>
+            </UnorderedList>
+          </ListItem>
+        </UnorderedList>
+      </td>
+      <td>No</td>
+    </tr>
+  </tbody>
+</table>
+
+---
+
+## Usage Guide
+
+### API
+
+#### Installation
+
+```bash
+yarn add @twilio-paste/form
+```
+
+#### Usage
+
+```jsx
+import {FormTextArea, FormLabel, FormHelpText} from '@twilio-paste/form';
+
+<FormLabel htmlFor="foo" required>foo</FormLabel>
+<FormTextArea aria-describedby="foo_text" id="foo" name="foo" placeholder="foo" required />
+<FormHelpText id="foo_text">Please enter some text</FormHelpText>
+```
+
+#### FormTextArea Props
```

Comment:
You might want to call out here that any valid attribute for an html textarea is also supported.

Same with label and helptext too

Reply:
I've added a snippet of text above each prop table.